frontend/app/routes/_protected+/_layout.tsx

```diff
@@ -14,16 +14,16 @@ export function ErrorBoundary() {
     switch (error.status) {
       // TODO :: GjB :: handle other status codes
       default:
-        return <ServerError error={error} />;
+        return <ServerError error={error} showAccountMenu={true} />;
```

Comment:
[Props Default to “True”](https://legacy.reactjs.org/docs/jsx-in-depth.html#props-default-to-true)

Reply:
I thought we agreed in a previous PR that we would be explicit when these sort of props were used. I'll change it.

Reply:
Fixed

Reply:
> I thought we agreed in a previous PR that we would be explicit when these sort of props were used. I'll change it.

Well, the absence of it "means" `false` even though it's `undefined`. The presence of it without an explicit value set it to `true`.